actstream/registry.py

```diff
@@ -57,7 +57,7 @@ def is_installed(model_class):
     Returns True if a model_class is installed.
```

Comment:
comments should also be adjusted, i think

Reply:
That part is still correct. The method will return True is the model class is installed in Django. If the app is not in the INSTALLED_APPS of Django this will return False. That is also the behavior before.